.github/workflows/test-upstream.yml

```
@@ -69,7 +69,6 @@ jobs:
           miniforge-variant: Mambaforge
           use-mamba: true
           python-version: ${{ matrix.python }}
-          channel-priority: strict
```

Why is strict priority dropped?

---

Setting strict channel priority raises an error when we attempt to update the `conda-forge` Dask packages to `dask/label/dev` - [here](https://github.com/dask-contrib/dask-sql/runs/6457470189?check_suite_focus=true) is an example of those failures.

---

The channels need to be [specified here]( https://github.com/conda-incubator/setup-miniconda#example-4-conda-options ) too

```suggestion
          channel-priority: strict
          channels: dask/label/dev,conda-forge
```

Would then drop the channel from the `mamba install` command. It may also be that command can be dropped if `dask` is in the environment file

---

Yeah good point - I think I can probably get away with setting the channels in a conditional for the other workflow as well